src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,20 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
+    public BulldozerProducer() {
+    }
+
+    @Override
+    public List<Machine> get() {
```

Comment:
The return type of the `get()` method should be `List<Bulldozer>` instead of `List<Machine>` to match the `MachineProducer<Bulldozer>` interface.